python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -851,19 +851,30 @@ async def _take_data(
         )
         exposures_done: asyncio.Future = asyncio.Future()
 
+        group_id = None
```

Comment:
I think group_id has to be a str value for the cmd expose in the fiberspectrograph (https://ts-xml.lsst.io/sal_interfaces/FiberSpectrograph.html#groupid) and electrometer (https://ts-xml.lsst.io/sal_interfaces/Electrometer.html#electrometer-commands-startscandt-groupid), so you should probably set this up to default to an empty string. 

You could do something like this:
`group_id = exposure_metadata.get("group_id", "")

if group_id == "":
    self.log.warning(
        "No Group ID for Electrometer/Fiber Spectrograph exposures. Continuing"
    )
`